buildscripts/setup.gradle

```diff
@@ -39,6 +43,80 @@ task renameAppPackage(type: Copy) {
     }
 }
 
+task renameMviPackage(type: Copy) {
+    description "Renames the template package in the mvi module."
+    group null
+
+    def newPackageAsDirectory = renameConfig.newPackage.replaceAll("\\.", "/")
+    def startingDirectory = "${rootDir}/mvi/src/main/java/${renameConfig.templateName}/mvi"
+    def endingDirectory = "${rootDir}/mvi/src/main/java/${newPackageAsDirectory}/mvi"
+
+    def startingPackage = "${renameConfig.templateName}.mvi"
+    def endingPackage = "${renameConfig.newPackage}.mvi"
+
+    from(startingDirectory)
+    into(endingDirectory)
+
+    // Replace package statements
+    filter { line ->
+        line.replaceAll(
+                "package $startingPackage",
+                "package $endingPackage"
+        )
+    }
+
+    // Replace import statements
+    filter { line ->
+        line.replaceAll(
+                "import $startingPackage",
+                "import $endingPackage"
+        )
+    }
+
+    doLast {
+        delete("${rootDir}/mvi/src/main/java/${renameConfig.templateName}")
+
+        ("git add $endingDirectory").execute()
+    }
+}
+
+task deleteMviModule {
+    description "Deletes MVI module and removes from settings.gradle.kts"
+    group null
+
+    doLast {
+        delete("${rootDir}/mvi")
+
+        def settingsFile = new File("${rootDir}/settings.gradle.kts")
```

Comment:
As we add more modules, we may want to have a `removeModuleFromSettingsGradle` task, so we can just pass in the module name and not duplicate this code. 

Similarly, may want a `removeModuleFromAppGradle` task that does the same.